code/modules/surgery/organs/augments_arms.dm

```diff
@@ -216,6 +223,15 @@
 		var/obj/item/assembly/flash/armimplant/F = locate(/obj/item/assembly/flash/armimplant) in items_list
 		F.I = src
 
+/obj/item/organ/cyberimp/arm/flash/Extend()
+	. = ..()
+	holder.set_light_range(7)
+	holder.set_light_on(TRUE)
+
+/obj/item/organ/cyberimp/arm/flash/Retract()
+	holder.set_light_on(FALSE)
+	..()
```

Comment:
```
	return ..()
```
Do you have a reason not to preserve the parent call return?

Reply:
It was an oversight, fixed now.